Add Documentation for AgentChat (#3635)
microsoft/autogen: python/packages/autogen-core/docs/src/agentchat-user-guide/index.md

myst:
  html_meta:
    "description lang=en": |
      User Guide for AgentChat, a high-level api for AutoGen
---

# AgentChat

> AgentChat is a high-level framework for building multi-agent applications with AI agents, utilizing the underlying `autogen-core` package. For beginner users, AgentChat is the recommended starting point. For advanced users, `autogen-core` provides more flexibility and control over the underlying components.

Reviewer: ```suggestion
AgentChat is a high-level package for building multi-agent applications built on top of the `autogen-core` package. For beginner users, AgentChat is the recommended starting point. For advanced users, `autogen-core` provides more flexibility and control over the underlying components.
```

> AgentChat aims to provide intuitive defaults, such as Agents with preset behaviors and Teams with predefined communication protocols, to simplify building multi-agent applications.

Reviewer: ```suggestion
AgentChat aims to provide intuitive defaults, such as **Agents** with preset behaviors and **Teams** with predefined communication protocols, to simplify building multi-agent applications.
```
